core/src/test/scala/unit/kafka/log/LogManagerTest.scala

```diff
@@ -242,7 +243,11 @@ class LogManagerTest {
   def testRecoveryDirectoryMappingWithRelativeDirectory() {
     logManager.shutdown()
     logDir = new File("data" + File.separator + logDir.getName)
-    logDir.mkdirs()
+    try {
+          Files.createDirectory(logDir.toPath())    
+      } catch {
+        case e: IOException => //this is good
```

Comment:
Yes, the exception causes the test to fail. Maybe I should leave the mkdirs calls as is in the tests...what do you think?